JSDemos/Demos/TabPanel/Overview/Angular/app/app.component.css

```diff
@@ -1,45 +1,122 @@
-::ng-deep .tabpanel-item {
-  height: 200px;
-  -webkit-touch-callout: none;
-  -webkit-user-select: none;
-  -khtml-user-select: none;
-  -moz-user-select: none;
-  -ms-user-select: none;
-  user-select: none;
-  padding-left: 25px;
-  padding-top: 55px;
+::ng-deep .tabpanel-demo {
+  display: flex;
+  height: 100%;
+  min-height: 450px;
 }
 
-::ng-deep .mobile .tabpanel-item {
-  padding-top: 10px;
+::ng-deep .widget-container {
+  display: inline;
+  flex-grow: 1;
+  min-width: 450px;
+  padding: 16px 32px;
 }
 
-::ng-deep .tabpanel-item > div {
-  float: left;
-  padding: 0 85px 10px 10px;
+::ng-deep .dx-theme-material .dx-tabpanel {
+  border-radius: 8px;
+  overflow: hidden;
 }
 
-::ng-deep .tabpanel-item p {
-  font-size: 16px;
-  margin: 0;
+::ng-deep .dx-tabs-vertical {
+  min-width: 120px;
 }
 
-::ng-deep .item-box {
-  font-size: 16px;
-  margin: 15px 0 45px 10px;
+::ng-deep .dx-multiview-item-content {
+  padding: 24px;
 }
 
 ::ng-deep .options {
+  display: inline-flex;
+  flex-direction: column;
+  flex-shrink: 0;
+  box-sizing: border-box;
+  width: 300px;
   padding: 20px;
   background-color: rgba(191, 191, 191, 0.15);
-  margin-top: 20px;
+}
+
+::ng-deep .tabpanel-item .task-item:not(:last-child) {
```

Comment:
Can we use 'gap' property for '.task-item' collection? In this case we should not think about last item

Reply:
[b47c4be5bc56d7d1d81ce5aa0c922116cd6b34b7]